src/react/Story.js

```diff
@@ -7,13 +7,15 @@ import { _clientAPI } from '../shim'
 export default class Story extends Component {
   static propTypes = {
     kind: PropTypes.string.isRequired,
-    name: PropTypes.string.isRequired
+    name: PropTypes.string.isRequired,
+    usageInstructions: PropTypes.node
```

Comment:
i really don't think `usageInstructions` belongs on a `Story`.

this would be better handled by the custom story template, no?